Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Core/RepeaterNodeable.ScriptCanvasNodeable.xml

```diff
@@ -10,17 +10,17 @@
            Category="Nodeables"
            GeneratePropertyFriend="True"
            Namespace="ScriptCanvas"
-           Description="Repeats the output signal the given number of times using the specified delay to space the signals out">
+           Description="Repeats the output signal the given number of times using the specified delay to space the signals out.">
 
 
       <!-- Input tag is for an execution input that has optional data (parameters) -->
       <Input Name="Start" Description="">
         <Parameter Name="Repetitions" Type="Data::NumberType" DefaultValue="0.0" Description="How many times to repeat."/>
-        <Parameter Name="Interval" Type="Data::NumberType" DefaultValue="0.0" Description="The Interval between repetitions."/>
+        <Parameter Name="Interval" Type="Data::NumberType" DefaultValue="0.0" Description="The Interval between repetitions. If zero, all repititions execute immediately, before On Start"/>
```

Comment:
Before On Start? is that the desired/expected behavior? should the default be non-zero?